Contributing.md

```diff
@@ -0,0 +1,47 @@
+# How to Contribute
```

Comment:
I wonder if you want to rename these files (CONTRIBUTING, INSTRUCTIONS) with all caps to match the README, USAGE, SETUP convention.

Reply:
This may sound ridiculous but I was trying to make them look more approachable and less technical-- we can definitely change it for consistency, though!

Reply:
I'd probably make them consistent, but it's your call :)

Reply:
Updated in 936df3a !